.github/ISSUE_TEMPLATE/feature---enhancement-request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature / Enhancement request
+about: Describe what you think is missing or could be improved in OpenRA.
+title: ''
+labels: ''
```

Comment:
'Feature' or 'Idea/Wishlist' maybe?

Reply:
I thought it might be better if this was decided per ticket individually like now.